### DISCUSSION TITLE
Does Navidrome constantly forget its login?

### DISCUSSION THREAD

**Discussion by yllekz** [navidrome/navidrome#3101] [General]
Hi folks, trying to figure something out. I've noticed when I use Navidrome in the browser, be it Safari, Firefox, etc., it seems that the login isn't remembered for very long, and if I close the window out and relaunch it, I have to log in again. Is there a setting to tune this?

**Comment by deluan** [comment]
There is a session timeout. If you don't use the web UI for a certain period of time (default `24h`), you will be automatically logged off.

You can change this value using the [`SessionTimeout`](https://www.navidrome.org/docs/usage/configuration-options/#:~:text=false-,SessionTimeout,-ND_SESSIONTIMEOUT) config option. For example, mine is set to `120h`
